app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -140,4 +153,25 @@ static String readJsonFile (Context context) {
         }
         return resp;
     }
+
+    public static void onLocationButtonClick(Chat fragment, Activity activity) {
+        maps.setMapsApiKey(BuildConfig.RC_MAPS_API_KEY)
+                 //.setSendButtonIconColor(Color.RED)
+                 //.setSendButtonIcon(R.drawable.bank_icon)
+                 //.setSendButtonBackgroundColor(Color.RED)
+                 //.setNavigationBarTitleColor(Color.RED)
+                 //.setNavigationBarBackgroundColor(Color.RED)
+                 // .setNavigationBarTitleFont(Typeface.DEFAULT_BOLD)
+                 // .setNavigationBarIconColor(Color.RED)
+                 // .setNavigationBarTitleSize((int) activity.getResources().getDimension(R.dimen.rc_navigation_bar_title_text_size_test))
+                 // .setButtonTextSize(12)
```

Comment:
consistency please 🙂

Reply:
about the space ?

Reply:
yes, you can choose whichever you prefer but we need consistency

Reply:
done